storage/redis_controller.go

```diff
@@ -17,9 +17,8 @@ type RedisController struct {
 
 	redisUp      atomic.Value
 	disableRedis atomic.Value
-
-	ctx       context.Context
-	reconnect chan struct{}
+	reconnect    chan struct{}
+	ctx          context.Context
```

Comment:
If you can, just revert these lines to keep the diff smaller, it's a no-op change

Reply:
done